components/locale.js

```diff
@@ -61,28 +62,72 @@ function getInitialLocaleProps({req} = {}) {
   return Promise.resolve({locale})
 }
 
-// eslint-disable-next-line complexity
 function getContent(locale, options) {
-  const {page, component, example} = options
-  try {
-    const localePath = locale === fallbackLocale ? '' : `${locale}/`
-    // because how webpack resolves these for the bundle, we need
-    // to use a statically relative path, otherwise this could
-    // be much simpler. Sigh...
-    if (page) {
-      return require(`../pages/${page}/content/${localePath}index.js`)
-    } else if (component) {
-      return require(`../components/content/${localePath}${component}`)
-    } else if (example) {
-      return require(`../examples/content/${localePath}${example}`)
-    } else {
-      throw new Error('page or component required to get content')
+  const fallbackContent = content('', options)
+  if (locale === fallbackLocale) {
+    return fallbackContent
+  }
+  const localeContent = content(`${locale}/`, options)
+  return mergeTranslations(fallbackContent, localeContent)
+}
+
+function mergeTranslations(fallbackContent, localeContent) {
+  if (!localeContent || localeContent === {}) {
```

Comment:
`localeContent === {}` will never be true. Maybe we could do: `Object.keys(localeContent).length === 0` :)